src/tokens/RwaTokenFactory.sol

```diff
@@ -0,0 +1,102 @@
+/* prettier-disable */
```

Comment:
Oh, this is important: since it's a new file, we should probably use NatSpec to document it.